package.json

```diff
@@ -32,9 +32,12 @@
     "caver-js": "^1.8.2",
     "debounce": "^1.2.1",
     "debug": "^4.3.4",
+    "dompurify": "^2.3.10",
+    "github-markdown-css": "^5.1.0",
     "graphql": "^16.5.0",
     "graphql-tag": "^2.12.6",
     "lodash": "^4.17.21",
+    "moment": "^2.29.4",
```

Comment:
I'd strongly recommend using `luxon` instead.

Anyway, both libraries are really heavy. Can we go without them at all?

Reply:
I replaced moment with dayjs. What do you think?

Reply:
Looks nice!